homeassistant/components/advantage_air/sensor.py

```diff
@@ -138,9 +142,10 @@ def icon(self):
 
 
 class AdvantageAirZoneTemp(AdvantageAirEntity, SensorEntity):
-    """Representation of Advantage Air Zone wireless signal sensor."""
+    """Representation of Advantage Air Zone temperature sensor."""
 
     _attr_native_unit_of_measurement = TEMP_CELSIUS
+    _attr_device_class = DEVICE_CLASS_TEMPERATURE
     _attr_state_class = STATE_CLASS_MEASUREMENT
     _attr_icon = "mdi:thermometer"
```

Comment:
You can't have an icon when you have a device class.

Reply:
Fixed.